R/Reactant/build_tarballs.jl

```diff
@@ -423,7 +466,7 @@ augment_platform_block="""
     """
 
 # for gpu in ("none", "cuda", "rocm"), mode in ("opt", "dbg"), platform in platforms
-for gpu in ("none", "cuda"), mode in ("opt", "dbg"), cuda_version in ("none", "12.4", "12.6", "12.8"), platform in platforms
+for gpu in ("none", "cuda", "rocm"), mode in ("opt", "dbg"), cuda_version in ("none", "12.4", "12.6", "12.8"), platform in platforms
 
```

Comment:
Like https://github.com/JuliaPackaging/Yggdrasil/pull/11820#discussion_r2271404066, while doing tests it'd be much better to ***only*** build the platforms to test
```suggestion
    gpu != "rocm" && continue
```